lxd/instance/instance_utils.go

```diff
@@ -128,6 +128,22 @@ func ValidConfig(sysOS *sys.OS, config map[string]string, expanded bool, instanc
 }
 
 func validConfigKey(os *sys.OS, key string, value string, instanceType instancetype.Type) error {
+	// Check if the key is a valid prefix and whether or not it requires a subkey.
+	knownPrefixes := append(instancetype.ConfigKeyPrefixesAny, instancetype.ConfigKeyPrefixesContainer...)
```

Comment:
can this knownPrefixes definition be reduced in scope by moving it inside the if below?

Reply:
oh i see you use it in next commit